book/src/dev/rfcs/0004-asynchronous-script-verification.md

```diff
@@ -426,10 +426,12 @@ ancestors are downloaded.
 These optimisations can be delayed until after the initial implementation is
 complete, and covered by tests:
 
+<!-- TODO: address the three items below since #2455 is closed. -->
+
 - Should we stop storing heights for non-coinbase UTXOs? (#2455)
 
 - Should we avoid storing any extra data for UTXOs, and just lookup the coinbase
-  flag and height using `outpoint.hash` and `tx_by_hash`? (#2455)
+  flag and height using `outpoint.hash` and `tx_loc_by_hash`? (#2455)
 
 - The maturity check can be skipped for UTXOs from the finalized state,
 because Zebra only finalizes mature UTXOs. We could implement this
```

Comment:
I think we can delete all these.

We only store `transparent::Output`s in `utxo_by_out_loc`, so the height and extra data changes are either done or obsolete:
https://github.com/ZcashFoundation/zebra/blob/main/book/src/dev/rfcs/0005-state-updates.md#rocksdb-data-structures

The maturity check can't be skipped, because:
1. The non-finalized state starts at less than 100 blocks from the finalized tip, and
2. Zebra can roll back the non-finalized state to a tip that is less than 100 blocks from the finalized tip